pkg/handler/v2/external_cluster/external_cluster.go

```diff
@@ -303,6 +345,11 @@ type deleteClusterReq struct {
 	// in: path
 	// required: true
 	ClusterID string `json:"cluster_id"`
+	// The action is used to check if to delete a externalcluster or a actual cluster
+	// in: header
+	// name: Action
+	// Possible values: DeleteExternalCluster, DeleteProviderCluster
+	Action string `json:"action"`
```

Comment:
Maybe add here that by default, the behavior is to just delete the External cluster object.

Also not sure if  the difference for DeleteExternalCluster and DeleteProviderCluster are clear enough. Maybe the Disconnect/Delete that you use in code is less ambigous